Platform/ElkhartlakeBoardPkg/Script/StitchIfwiConfig.py

```diff
@@ -79,32 +79,32 @@ def get_component_replace_list():
       ('IFWI/BIOS/TS1/ACM0',      'Input/acm.bin',        'dummy',    '',                            ''),
     ]
 
-    if os.path.exists('IPFW/PreOsChecker.bin'):
+    if os.path.exists('IPFW/PreOsChecker.bin') and os.path.exists('IFWI/BIOS/NRD/IPFW/POSC'):

```

Comment:
"IFWI/BIOS/NRD/IPFW/POSC" is not really file system file. it is the component path from flash path.